plugins/rtmp-services/data/services.json

```diff
@@ -1145,6 +1145,28 @@
                 "max audio bitrate": 192
             }
         },
+        {
+            "name": "EventLive.pro",
+            "alt_names": ["EventLive", "Event Live"],
+            "servers": [
+                {
+                    "name": "Default",
+                    "url": "rtmp://go.eventlive.pro/live"
+                }
+            ],
+            "recommended": {
+                "keyint": 2,
+                "profile": "baseline",
```

Comment:
Enforcing baseline profile will produce very low quality, is your audience really using such ancient devices that they cannot handle main or high profile?